services/common/src/components/common/RenderAutoComplete.js

```diff
@@ -0,0 +1,76 @@
+import React from "react";
```

Comment:
Hmmm, this might get a bit sticky.  Tara has spent a bunch of time moving all of the form components into the common package and converting them to .tsx.  Going to have to coordinate merging these I think.

Reply:
Removed unnecessary props.